tests/ui/crashes/procedural_macro.rs

```diff
@@ -1,11 +0,0 @@
-#[macro_use]
-extern crate clippy_mini_macro_test;
-
-#[deny(warnings)]
-fn main() {
-    let x = Foo;
-    println!("{:?}", x);
-}
-
-#[derive(ClippyMiniMacroTest, Debug)]
-struct Foo;
```

Comment:
Deleted this for convenience since /crashes has a separate auxilary folder. The test is redundant and history shows no evidence that it ever produced an ICE (see #3743).

Reply:
I also can't find the reason why this exists. Probably to check that the derive doesn't produce an ICE. But since we use this in other tests too, deleting this test should be fine.